charts/index.yaml

```diff
@@ -45,17 +45,17 @@ entries:
   cromwell:
   - apiVersion: v2
     appVersion: 68-8e12ab5
```

Comment:
The `appVersion` now appears to be out of date

Reply:
I changed this in multiple places with the latest commit. Did I miss any instances?

Reply:
I don't think you should be updating the index.yaml file directly (I've made that mistake myself twice!). And I don't think you want to change the Cromwell version ID for older helm file versions… those were legitimately made with a different release of Cromwell.